### ISSUE TITLE
New Quests [ID: 10]: given Quests are always at Duraka Hills 

### ISSUE THREAD

**Issue by Gumpiboi** [thetrueoneshots/CubeMegaMod#15]
A friend and I were playing together and we noticed that all the quest scrools we get from npcs are at the Duraka Hills Region.
But the Quest counter does work regardless. When a mob is killed the specific quest counter increases by 1.
I haven't found out what happens when a quest gets completed, because i haven't finished one.

**Comment by gijsgroenewegen** [comment]
Yes, all quests do have that region (probably the 0,0 region). Quest should complete normally (giving you gold) and get removed from the inventory after completion.
